docs/sources/write/front-matter/index.md

```diff
@@ -68,38 +69,65 @@ The correct way to use aliases depends on whether the project is versioned or no
+| Variable        | Page                                                                 |
+| --------------- | -------------------------------------------------------------------- |
+| _`SOURCE`_      | `/docs/grafana/latest/alerting/silences/`                            |
+| _`DESTINATION`_ | `/docs/grafana/latest/alerting/manage-notifications/create-silence/` |
 
-- The relative alias `./` refers to the page`/docs/grafana/latest/alerting/manage-notifications/` because that is the directory containing the page `/docs/grafana/latest/alerting/manage-notifications/create-silence/`.
-- The relative alias `../` refers to the page `/docs/grafana/latest/alerting/`.
-- The relative alias `../silences/` refers to the page `/docs/grafana/latest/alerting/silences/`.
+- The current directory element (`.`) refers to the directory containing _`DESTINATION`_.
 
-Therefore, with the alias `../silences/` in the source file for the page `/docs/grafana/latest/alerting/manage-notifications/create-silence/`, Hugo will create a redirect page at `/docs/grafana/latest/alerting/silences/`.
+  In this example, the page `/docs/grafana/latest/alerting/manage-notifications/`.
 
-To redirect the page `/docs/grafana/latest/alerting/unified-alerting/` to `/docs/grafana/latest/alerting/` you must add a relative alias in the source file for `/docs/grafana/alerting/` containing the relative alias to `/docs/grafana/latest/alerting/unified-alerting/`.
+- The parent directory element (`..`) refers to the parent directory of the current directory element.
 
-- The relative alias `./` refers to the page `/docs/grafana/latest/` because that is the directory containing the page `/docs/grafana/latest/alerting/`.
-- The relative alias `./alerting/` refers to the page `/docs/grafana/latest/alerting/` which is the page itself.
-- The relative alias `./alerting/unified-alerting/` refers to the page `/docs/grafana/latest/alerting/unified-alerting/`.
+  In this example, the page `/docs/grafana/latest/alerting/`.
 
-Therefore, with the alias `./alerting/unified-alerting/` in the source file for the page `/docs/grafana/latest/alerting/`, Hugo will create a redirect page at `/docs/grafana/latest/alerting/unified-alerting/`.
+Therefore, with the alias `../silences/` in the source file for the page `/docs/grafana/latest/alerting/manage-notifications/create-silence/`, Hugo creates a redirect page at `/docs/grafana/latest/alerting/silences/`.
+
+<hr/>
+
+The following example redirects _`SOURCE`_ to _`DESTINATION`_ using the pages:
+
+| Variable        | Page                                              |
+| --------------- | ------------------------------------------------- |
+| _`SOURCE`_      | `/docs/grafana/latest/alerting/unified-alerting/` |
+| _`DESTINATION`_ | `/docs/grafana/latest/alerting/`                  |
+
+- The current directory element (`.`) refers to the directory containing _`DESTINATION`_.
```

Comment:
This is confusing because there is no `.` in the table.   If you're giving an example, the example needs to include the `.` .